keyboards/nyquist/keymaps/skug/keymap.c

```diff
@@ -0,0 +1,177 @@
+ * |------+------+------+------+------+------|   |------+------+------+------+------+------|
+ * |      | Left | Down | Right|      |      |   |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|   |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |   |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+   +------+------+------+------+------+------|
+ * |      |      |      |  Del |      |      |   |      |      |      |      |      |      |
+ * `-----------------------------------------'   '-----------------------------------------'
+ */
+[ARRW] = LAYOUT( \
+  NO_HALF,  _______,  _______,  _______ ,  _______,  _______,      _______,  _______,  _______,  _______,  _______,  _______, \
+  _______,  _______,  KC_UP  ,  _______ ,  _______,  _______,      _______,  _______,  _______,  _______,  _______,  _______, \
+  _______,  KC_LEFT,  KC_DOWN,  KC_RIGHT,  _______,  _______,      _______,  _______,  _______,  _______,  _______,  _______, \
+  _______,  _______,  _______,  _______ ,  _______,  _______,      _______,  _______,  _______,  _______,  _______,  _______, \
+  _______,  _______,  _______,  KC_DEL  ,  _______,  _______,      _______,  _______,  _______,  _______,  _______,  _______  \
+),
+
+/* Adjust ()
+ * ,-----------------------------------------------------------------------------------.
+ * |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      | Reset|RGB Tg|RGB Md|Hue Up|Hue Dn|Sat Up|Sat Dn|Val Up|Val Dn|      |  Del |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |      |      |Aud on|Audoff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|      |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] =  LAYOUT( \
+  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12, \
+  _______, RESET  , RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI, RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, _______, KC_DEL, \
+  _______, _______, _______, AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, QWERTY,  COLEMAK, DVORAK,  _______, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______ \
+)
+
+};
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Could you remove this function?  

It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer